modules/apps/portlet-configuration/portlet-configuration-web/src/main/java/com/liferay/portlet/configuration/web/internal/portlet/configuration/icon/PermissionsPortletConfigurationIcon.java

```diff
@@ -174,9 +173,6 @@ public boolean isUseDialog() {
 	@Reference
 	private Language _language;
 
-	@Reference
-	private PortletLocalService _portletLocalService;
```

Comment:
@W2jx `PortletPermissionUtil` in this class will finally call `PortletLocalServiceUtil`, let's hold this one.